Source/Scene/Scene.js

```diff
@@ -1444,7 +1493,7 @@ define([
 
         if (environmentState.isSunVisible) {
             environmentState.sunDrawCommand.execute(context, passState);
-            if (scene.sunBloom) {
+            if (scene.sunBloom && !scene._useWebVR) {
```

Comment:
Yes, whenever bloom is enabled and the sun is in view, the first viewport is either not rendered or clear before the second viewport is rendered.